website/content/docs/nia/terraform-modules.mdx

```diff
@@ -31,21 +31,23 @@ See below sections for more information on [defining source input](#source-input
 
 ### Source Input
 
-A source input allows for a task to satisfy the input requirements defined by the Terraform Module’s [input variables](https://www.terraform.io/docs/language/values/variables.html), and is configured alongside a task’s condition. Both the source input and condition define objects to be monitored, but for differing reasons.
+CTS monitors defined objects to provide source input to a task. This source input satisfies the requirements defined by the task's Terraform Module's [input variables](https://www.terraform.io/docs/language/values/variables.html).
```

Comment:
I'm not sure I understand what is being stated here.  What are object in reference to?

Reply:
Thanks for asking! The objects refer to the Consul objects that CTS monitors for change. In the case of source input, these Consul objects are converted into variables that are needed by the task's Terraform Module. Maybe I could change it to:

> A task monitors Consul objects that are specified by the task's configuration. The Consul objects can be used for the source input that satisfies the requirements defined by the task's Terraform Module's input variables.

Welcome your input!

Reply:
Yea - as of now Consul object can be services instances, service registration, or consul-kv pair. We're planning to add additional "objects". Open to new terminology!

Reply:
Gotcha, that helps a lot. What we can do is add an entry in the [Consul glossary page](https://www.consul.io/docs/install/glossary). Then when we use the term Consul objects we can link to that spot in the glossary. At least the first time we use the term.

Reply:
Interesting! What do you think about adding the entry in the [CTS glossary section](https://www.consul.io/docs/nia#glossary) instead? I think this terminology will only be used by CTS so was thinking about avoiding impacting Consul docs. I'll defer to your thoughts!

Reply:
Oh, I see. That's good to know. What are your thoughts on:
- I remove "Consul" from the sentence. Thinking about this some more, we may expand beyond monitoring Consul. It might be better to be better to leave object more generic. (Not sure if this reduces reader confusion)
- I create a task to combine the CTS glossary with Consul's glossary. I can raise it with my team in our next week's meeting.

Reply:
At the very least, these glossary items should be in the main glossary page. As for leaving it generic, that's what is confusing. As a practitioner, I should at least have an idea of what these objects are. Otherwise, it hard for me to understand how to leverage the configuration options if I don't know what is in scope.

Reply:
Thanks for sharing the perspective on leaving it generic. I'll keep "Consul". I'd like to initially add the "Consul object" terminology in the CTS glossary section for this PR. Then have a separate PR to combine the CTS glossary over to the Consul glossary.

I'd like to do a separate PR since I don't think the glossary will be a straight copy-over. Many of the entries are written within the context of CTS and will need some rework. For example, the entry for "Terraform Module" was intended to give CTS readers a quick definition, but it could look awkward as is in the Consul glossary.